libs/core/src/lib/components/wrapper-component.ts

```diff
@@ -213,6 +219,10 @@ export abstract class ReactWrapperComponent<TProps extends {}> implements AfterV
       return (context: TContext) => htmlRenderer.render(context);
     }
 
+    if (input instanceof RenderOverride) {
+      return undefined;
```

Comment:
why not `return input.renderer`?

Reply:
How I wrote in Issue #72, these changes help me to set the styles 
This is the code example:

```
private overrideRenderRow(props: IDetailsRowProps, defaultRender: (props: IDetailsRowProps) => JSX.Element): JSX.Element {
  props.styles = MY_STYLES;
  return defaultRender(props);
}
```

Reply:
Sounds reasonable - you want to expose the native JSX to Angular-land then. This should be fine for simple cases, since JSX in Angular is something I’d like to avoid (although possible of course).

However, I suggest we change the API, to avoid the boilerplate of creating a class (which would really only have one instance), if possible.
I want to think about an API further, and see if we can somehow avoid exposing JSX to the final user (i.e. the developer using `@angular-react/fabric`) - perhaps allowing to pass a function that accepts `TProps` and returns `TProps`, which will then be fed to the `defaultRenderer` that `office-ui-fabric-react` exposes. If you have an idea for an API - I’m open for suggestions, and how it would work.